scripts/sanity_chk/sanitylib.py

```diff
@@ -2058,6 +2058,7 @@ def process(self, message):
                 self.instance.reason = "Build Failure"
                 pipeline.put({"op": "report", "test": self.instance})
             else:
+                self.instance.status = "passed"
```

Comment:
If this fix is for build_only instances maybe it better to add a specific check for this before filling the status? E.g.:
```
else:
    if self.instance.build_only:
        self.instance.status = "passed"
```

Reply:
This status should be set regardless of build_only, we set error above but we set no status when the build is successful.

Reply:
addressed